week-1/Homework/mandatory/5-magic-8-ball.js

```diff
@@ -42,20 +42,75 @@ My sources say no.
 Outlook not so good.
 Very doubtful.
 */
+function getNumber() {
+  let pick_random = Math.floor(Math.random() * 4) + 1;
+  return pick_random;
+}
+console.log(getNumber());
 
 // This should log "The ball has shaken!"
 // and return the answer.
-function shakeBall() {}
+function shakeBall() {
+  
+  switch (randomNumber) {
+    case 1:
+      answer = "It is certain.";
```

Comment:
Your magic 8 ball works well, but it only has one answer for each sentiment! This passes our tests, because testing randomness is hard (good lesson for the future!), but it would be great to expand this code to be able to return all 20 possible answers. Can you give it a go?

Next week we're going to learn some things about arrays and objects which will help us do this with less code, too, but it's worth practicing with what you already know :)